cases/src/misc.rs

```diff
@@ -135,6 +135,12 @@ pub fn shrink_to_imm(x: u64) -> i8 {
     }
 }
 
+// 0 to 15
+pub fn shrink_to_imm_u(x: u64) -> u8 {
```

Comment:
The `imm` has 5 bits, but 16 only takes 4.

Reply:
This function is only used for immediate instruction's testcase. Each intruction needs to be manually coded, so here I only use 0~15.

Reply:
I'm sorry, I misunderstood this a bit before, I'm working on it.